custom_components/hacs/translations/en.json

```
@@ -67,8 +67,8 @@
             }
         },
         "removed": {
-            "title": "Repository removed from HACS",
-            "description": "{name} has been removed from HACS for {reason} visit the [HACS Panel](/hacs/repository/{repositry_id}) to remove it."
+            "title": "Repository or component removed from HACS",
+            "description": "'{name}' has been removed from HACS, because {reason}. Please visit the [HACS Panel](/hacs/repository/{repositry_id}) to remove it locally."
```

```suggestion
            "description": "'{name}' has been removed from HACS, because {reason}. Please visit the [HACS Panel](/hacs/repository/{repositry_id}) to remove it."
```
It can not be anything else.

---

It simply sounds better and more consistent this way: "The repository online is gone, so you should remove locally."